CCEv3 - Parameter Subnet-ID

can you please check https://www.terraform.io/docs/providers/opentelekomcloud/r/cce_cluster_v3.html
opentelekomcloud_cce_cluster_v3

there is a parameter: 
_subnet_id - (Required) The ID of the subnet used to create the node. Changing this parameter will create a new cluster resource._

This is very misleading, as you have to use the **Network_ID** and not the Subnet_ID. Please rename the parameter or adjust the description. 

### Terraform Version
Terraform v0.12.24
+ provider.opentelekomcloud v1.18.0

### Affected Resource(s)
opentelekomcloud_cce_cluster_v3

### Terraform Configuration Files
resource "opentelekomcloud_cce_cluster_v3" "cluster_1" {
  name                   = "cluster"
  cluster_type           = "VirtualMachine"
  flavor_id              = "cce.s1.small"
  vpc_id                 = var.vpc
  subnet_id              = var.subnet
  container_network_type = "overlay_l2"
  authentication_mode    = "rbac"
  description            = "Create cluster"
}


### Debug Output
2020-06-26T07:59:29.000+0200 [INFO]  plugin: configuring client automatic mTLS
2020-06-26T07:59:29.036+0200 [DEBUG] plugin: starting plugin: path=/root/terraform/cce/ccev2/hermes/.terraform/plugins/linux_amd64/terraform-provider-opentelekomcloud_v1.18.0_x4 args=[/root/terraform/cce/ccev2/hermes/.terraform/plugins/linux_amd64/terraform-provider-opentelekomcloud_v1.18.0_x4]
2020-06-26T07:59:29.039+0200 [DEBUG] plugin: plugin started: path=/root/terraform/cce/ccev2/hermes/.terraform/plugins/linux_amd64/terraform-provider-opentelekomcloud_v1.18.0_x4 pid=1807
2020-06-26T07:59:29.039+0200 [DEBUG] plugin: waiting for RPC address: path=/root/terraform/cce/ccev2/hermes/.terraform/plugins/linux_amd64/terraform-provider-opentelekomcloud_v1.18.0_x4
2020-06-26T07:59:29.047+0200 [INFO]  plugin.terraform-provider-opentelekomcloud_v1.18.0_x4: configuring server automatic mTLS: timestamp=2020-06-26T07:59:29.047+0200
2020-06-26T07:59:29.080+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.18.0_x4: plugin address: network=unix address=/tmp/plugin493365084 timestamp=2020-06-26T07:59:29.080+0200
2020-06-26T07:59:29.080+0200 [DEBUG] plugin: using plugin: version=5
2020-06-26T07:59:30.037+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.18.0_x4: 2020/06/26 07:59:30 [INFO] Building Swift S3 auth structure
2020-06-26T07:59:30.037+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.18.0_x4: 2020/06/26 07:59:30 [INFO] Setting AWS metadata API timeout to 100ms
2020-06-26T07:59:30.057+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.18.0_x4: 2020/06/26 07:59:30 [INFO] AWS EC2 instance detected via default metadata API endpoint, EC2RoleProvider added to the auth chain
2020-06-26T07:59:30.057+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.18.0_x4: 2020/06/26 07:59:30 [INFO] Swift S3 Auth provider used: "StaticProvider"
opentelekomcloud_cce_cluster_v3.cluster_1: Creating...
2020/06/26 07:59:30 [DEBUG] opentelekomcloud_cce_cluster_v3.cluster_1: applying the planned Create change
2020-06-26T07:59:30.061+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.18.0_x4: 2020/06/26 07:59:30 [DEBUG] setting computed for "certificate_clusters" from ComputedKeys
2020-06-26T07:59:30.061+0200 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.18.0_x4: 2020/06/26 07:59:30 [DEBUG] setting computed for "certificate_users" from ComputedKeys
2020/06/26 07:59:30 [DEBUG] opentelekomcloud_cce_cluster_v3.cluster_1: apply errored, but we're indicating that via the Error pointer rather than returning it: Error creating opentelekomcloud Cluster: Internal Server Error
2020/06/26 07:59:30 [ERROR] <root>: eval: *terraform.EvalApplyPost, err: Error creating opentelekomcloud Cluster: Internal Server Error
2020/06/26 07:59:30 [ERROR] <root>: eval: *terraform.EvalSequence, err: Error creating opentelekomcloud Cluster: Internal Server Error

Error: Error creating opentelekomcloud Cluster: Internal Server Error

  on cce.tf line 1, in resource "opentelekomcloud_cce_cluster_v3" "cluster_1":
   1: resource "opentelekomcloud_cce_cluster_v3" "cluster_1" {


2020-06-26T07:59:30.729+0200 [DEBUG] plugin: plugin process exited: path=/root/terraform/cce/ccev2/hermes/.terraform/plugins/linux_amd64/terraform-provider-opentelekomcloud_v1.18.0_x4 pid=1807
2020-06-26T07:59:30.729+0200 [DEBUG] plugin: plugin exited

### Expected Behavior
cluster should be created

### Actual Behavior
Internal Server Error
![grafik](https://user-images.githubusercontent.com/67453937/85825242-05d99b00-b782-11ea-95a2-9d51ab6ec1b6.png)


### Steps to Reproduce
Set Subnet ID to the actual Subnet ID and not the required Network ID
1. `terraform apply`

--- a/website/docs/r/cce_cluster_v3.html.md
+++ b/website/docs/r/cce_cluster_v3.html.md
@@ -68,7 +68,7 @@ The following arguments are supported:
 
 * `vpc_id` - (Required) The ID of the VPC used to create the node. Changing this parameter will create a new cluster resource.
 
-* `subnet_id` - (Required) The ID of the subnet used to create the node. Changing this parameter will create a new cluster resource.
+* `subnet_id` - (Required) The Network ID of the subnet used to create the node. Changing this parameter will create a new cluster resource.
 
 * `highway_subnet_id` - (Optional) The ID of the high speed network used to create bare metal nodes. Changing this parameter will create a new cluster resource.
 

